<launch>

    <!-- <include file="$(find pointcloud_utils)/launch/shell_lidar_calibration.launch" /> -->

		<node pkg="pointcloud_utils" type="point_cloud_2_to_png.py" name="pc_grid_parser" required="true" output="screen" cwd="node">
      	   <param name="cloud_topic" 		value="/airsim_node/PhysXCar/lidar/VLP16_1" />
      	   <param name="target_frame" 		value="VLP16_1" />
      	   <param name="binary_mode" 		value="true" />
      	   <param name="publish_image" 		value="true" />
      	   <param name="save_image" 		value="true" />
		</node>
</launch>